extension-manifest-v3/src/pages/panel/components/container.js

```diff
@@ -0,0 +1,82 @@
+import { html } from 'hybrids';
+
+function updateShadow({ render }) {
+  const root = render();
+
+  const el = root.querySelector('#scroll');
+  const shadowTop = root.querySelector('.shadow.top');
+  const shadowBottom = root.querySelector('.shadow.bottom');
+
+  if (el.scrollHeight > el.clientHeight) {
+    shadowTop.classList.toggle('show', el.scrollTop !== 0);
+    shadowBottom.classList.toggle(
+      'show',
+      el.scrollTop + el.clientHeight < el.scrollHeight,
+    );
+  } else {
+    shadowTop.classList.remove('show');
+    shadowBottom.classList.remove('show');
+  }
+}
+
+export default {
+  _: {
```

Comment:
that's just a convention to manage the attached side effects, right?

How about using a symbol instead ? ;) jk

Reply:
This might be a missing feature of hybrids. There is no simple way to attach `connect/disconnect` callback directly to the `render/content` properties. They must be functions to work properly, so you must create another property with the `connect()` method. The value of this property is irrelevant in this case, so I just have used `_` for convenience. 

Symbols are not iterated using `Object.keys()`, and this is by design, as `[router.connect]` and other similar properties use symbols, to avoid generating properties on the host.